weblate_schemas/messages.py

```diff
@@ -0,0 +1,117 @@
+# Copyright © Michal Čihař <michal@weblate.org>
+#
+# SPDX-License-Identifier: MIT
+
+"""Message class implementation for Weblate Fedora Messaging."""
+
```

Comment:
```suggestion

from __future__ import annotations

```

Without that you end up with `TypeError: unsupported operand type(s) for |: 'type' and 'types.GenericAlias'` on Python 3.9.